__tests__/api.test.js

```diff
@@ -62,5 +62,57 @@ describe('app', () => {
             })

```

Comment:
Id like to see some error handling for this endpoint. What if we request an out of range review? What if we request a review with an id that isnt a number?